spec/controllers/users_controller_spec.rb

```diff
@@ -53,7 +70,7 @@
     it "sets @user" do
       set_current_user
       alice = Fabricate(:user)
-      get :show, id: alice.id
+      get :show, token: alice.token
```

Comment:
This test isn't passing. The problem is that the route is setup to take an `id` instead of `token`. You need to make sure the route, the test, and the controller are all shifted to using token instead of id. Also if you are going to do this you should modify the test above it so that it is trying the valid route instead of still trying to use the `id`.